src/map/script.c

```diff
@@ -9013,6 +9013,32 @@ BUILDIN(getgmlevel)
 	return true;
 }
 
+/// set the group ID of the player.
+/// setgroupid( <new group id>,{<account ID> or <Player Name>} )
+/// return 1 on success, 0 if failed.
+BUILDIN(setgroupid) {
+	TBL_PC* sd;
```

Comment:
oops, yes, its `TBL_PC *sd = NULL;`, thx

yes its possible to use `TBL_PC` in script.c file because it calls from map.h
https://github.com/HerculesWS/Hercules/blob/22d3fac0c9a9c39a4f3b544867ba1de4d8d2ecaa/src/map/map.h#L838